### ISSUE TITLE
Warning color not accessbile

### ISSUE THREAD

**Issue by DusanMilko** [carbon-design-system/carbon#2433]
For displaying errors the color for text is a red - #da1e28;
This red passes most accessibility on white background.

For displaying warnings the color for text is a yellow - #fdd13a;
This yellow **fails** all accessibility on white background.

Recommend using a yellow that passes on white background for atleast AA standard.
Or should warning text be displayed differently?

Also minor suggestion to updating the red to pass all accessibility.

**Comment by dakahn** [comment]
Great! Can you provide a test case where I can confirm the violations?

**Comment by aagonzales** [comment]
Are you using the red and yellow color for text? Can you post an image so we have a clear understanding of what you are referring to please.
